google/cloud/security/inventory/README.md

```diff
@@ -44,7 +44,50 @@ $ scripts/dev_inventory.sh
 ```
 
 ## How to Test
-Look at the test instruction in this [README](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/google/cloud/security/README.md#tests).
+Look at the test instruction in this [README].
+
+## How to Add New Resource to Inventory
+
+In general, it's straightforward to add new resource to inventory.
+The workflow is designed to be generic, with the main steps as:
+* define the database table schema to store the resource data
+* create a new pipeline worker, to process the new table and the table fieldnames
+* write a transformation to flatten the data returned by GCP APIs into a format
+that's ingestable by the database table
+
+Look at this PR for an [end-to-end example loading of a resource].
+
+Step-by-Step:
+1. Design the applicable database table schema.  This schema will reflect
+how data from GCP APIs will be flattened and stored in a database table.
```

Comment:
The sentence "This schema will reflect..." is confusing/seems unnecessary to me because don't all database schemas show what is going to be stored in the table? It seems to me that the real information we need to tell the user is something like:

"Design a new schema for the data you need to store in inventory. Each field of data should correspond to a column in the table schema."